README.md

```diff
@@ -4,7 +4,16 @@
   <br /><br /><br />
 </div>
 
-Tachyon is a ZKP accelerator using CUDA.
+# Overview
+
+**Tachyon** is a Modular ZK Backend, powered by GPU.
+
+## Design Goals
+
+1. General Purpose: We want a code that enables switch different proving scheme.
+2. Easy to use: Switching code should be easy.
+3. Blazing Fast: Of course, it should be fast.
```

Comment:
3. Blazing Fast: Tachyon's foremost requirement is speed, and not just any speed, but blazing speed! This entails Tachyon delivering exceptional performance on both CPU and GPU platforms.